0.1.52 appears to raise MSRV?

Building 0.1.52 appears to raise the MSRV (https://github.com/pyca/cryptography/actions/runs/3345289024/jobs/5540627366)

```
     Compiling iana-time-zone v0.1.52
       Running `rustc --crate-name iana_time_zone --edition=2018 /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/iana-time-zone-0.1.52/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts --crate-type lib --emit=dep-info,metadata,link -C opt-level=3 -C linker-plugin-lto -C overflow-checks=on --cfg 'feature="fallback"' -C metadata=a1f314261aac0584 -C extra-filename=-a1f314261aac0584 --out-dir /home/runner/work/cryptography/cryptography/src/rust/target/release/deps -L dependency=/home/runner/work/cryptography/cryptography/src/rust/target/release/deps --cap-lints allow`
  error[E0658]: the `unsafe_op_in_unsafe_fn` lint is unstable
    --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/iana-time-zone-0.1.52/src/lib.rs:10:1
     |
  10 | #![warn(unsafe_op_in_unsafe_fn)]
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     |
     = note: see issue #71668 <https://github.com/rust-lang/rust/issues/71668> for more information
```

this isn't documented in the changelog, and I don't think is intended?

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -7,7 +7,6 @@
 #![warn(missing_docs)]
 #![warn(rust_2018_idioms)]
 #![warn(trivial_casts, trivial_numeric_casts)]
-#![warn(unsafe_op_in_unsafe_fn)]
 #![warn(unused_qualifications)]
 #![warn(variant_size_differences)]
 

